src/DetailsView/Styles/detailsview.scss

```diff
@@ -1250,6 +1250,18 @@ div.insights-file-issue-details-dialog-container {
                         font-weight: $fontWeightSemiBold;
                     }
 
+                    .ms-Button-label {
```

Comment:
I noticed we're putting these styles in the 'issue-detail-outer-container'. Is this the container for the buttons in issue detail dialog?

Reply:
I added these styles to the `issue-detail-outer-container` to make the "Copy failure details" and "File issue" buttons in the details view consistent with the buttons in the details dialog

Reply:
I just added a screenshot to the description for this

Reply:
Moved the styles to `copy-issue-details-button` and `create-bug-button`